index.js

```diff
@@ -94,14 +94,23 @@ Mailgen.prototype.parseParams = function (params) {
     if (!body || typeof body !== 'object') {
         throw new Error('Please provide the `body` parameter as an object.');
     }
-    
+
     // Pass text direction to template
     body.textDirection = this.textDirection;
-    
+
     // Support for custom greeting/signature (fallback to sensible defaults)
     body.greeting = body.greeting || 'Hi';
     body.signature = body.signature || 'Yours truly';
 
+    function convertToArray(data) {
```

Comment:
Looks good, can you please move the function declaration outside of the `parseParams` function?